content/help/ops/setup/webhook/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Dynamic Admission Webhooks
+description: Describes Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+## Webhook configuration overview
```

Comment:
Here and elsewhere, you can just remove the first header in the docs. The site will already put your title has a header for the whole page, and we don't want the first piece of content to be another header.